test/nested-properties.js

```diff
@@ -25,18 +25,18 @@ test("dom node style", function (assert) {
 
     var rootNode = render(a)
     assert.equal(rootNode.style.border, style("border", "none"))
-    assert.equal(rootNode.style.className, style("className", "oops"))
+    assert.equal(rootNode.style["class-name"], style("className", "oops"))
     assert.equal(rootNode.style.display, style("display", "none"))
     var s1 = rootNode.style
     var equalNode = render(b)
     assert.equal(equalNode.style.border, style("border", "1px solid #000"))
-    assert.equal(equalNode.style.className, style("className", "oops"))
+    assert.equal(equalNode.style["class-name"], style("className", "oops"))
```

Comment:
I think this change is "bad". I don't think this change should happen at this level of the abstraction.